snuba/query/parsing.py

```diff
@@ -0,0 +1,15 @@
+class ParsingContext:
```

Comment:
This is likely to evolve as long as we change the process of parsing the query. Right now it does not do much, since the only goal is to split the parsing state from the body of the query.

Reply:
Yeah, as I was reading this, it seemed like it might be leading towards making things like the `column_expr` part of this class itself…